.github/workflows/build.yml

```
@@ -278,7 +278,7 @@ jobs:
             packages/*.sha256
 
   package-nuget:
-    needs: [ windows-builds, macos-build, macos-cross-build, macos-universal-package, docker-builds]
+    needs: [ windows-builds, macos-build, macos-cross-build, macos-universal-package, docker-builds, linux-musl-build]
```

This can be simplified since `macos-cross-build` and `macos-build` are already dependencies of `macos-universal-package` and you're also not using `docker-builds` anymore.

```suggestion
    needs: [ windows-builds, macos-universal-package, linux-musl-build]
```